lib/common/models/redis_store.rb

```diff
@@ -76,10 +76,14 @@ def self.delete(redis_key = nil)
       redis_namespace.del(redis_key)
     end
 
-    def save
+    def save(record_ttl = nil)
```

Comment:
so do we know that this mechanism will be used by anything else other than MVI RedisStores right now? If not, I don't think these changes ought to be made in RedisStore... I'm strongly in favor of keeping RedisStore simple and introducing complexity on an application by application basis. Is there any reason why we don't just override the save method on MVI?